### ISSUE TITLE
Plants don't grow without sun unless super glow glass has only 1 block of space (tested with tomatoes)

### ISSUE THREAD

**Issue by Poikilos** [Poikilos/Enliven#208]
Glow block and dirt must only have 1 air block between them
* only grows the plant straight under it
* happens with grass too

**Comment by slopsbucket** [comment]
Not a bug.

Deliberate and intentional game balance choice.

**Comment by poikilos** [comment]
That sounds reasonable. @HenryNautilus should I close this one or override on ENLIVEN?

**Comment by HenryNautilus** [comment]
meh I guess this can be closed. if it's meant to balance. though isn't corn a two block crop? I don't know cuz i haven't farmed it yet.

**Comment by HenryNautilus** [comment]
nevermind! corn can just grow THROUGH the glass.

**Comment by poikilos** [comment]
Yes, I was curious about those 2-block high plants. I'll look into making this configurable.

**Comment by poikilos** [comment]
Generally, the light level must be 13 to grow plants, though they can be set individually: <https://forum.minetest.org/viewtopic.php?f=3&t=11097>.
